File: Second Year/2nd Semester/Medii de Proiectare si Programare/Java/Laborator_1/src/main/java/domain/Participant.java
package domain;

public class Participant extends Entity<Integer>{
    private String first_name;
    private String last_name;
    private int points;

    public Participant(Integer integer, String first_name, String last_name, int points) {
        super(integer);
        this.first_name = first_name;
        this.last_name = last_name;
        this.points = points;
    }

    public String getFirst_name() {
        return first_name;
    }

    public void setFirst_name(String first_name) {
        this.first_name = first_name;
    }

    public String getLast_name() {
        return last_name;
    }

    public void setLast_name(String last_name) {
        this.last_name = last_name;
    }

    public int getPoints() {
        return points;
    }

    public void setPoints(int points) {
        this.points = points;
    }

    @Override
    public String toString() {
        return "Participant{" +
                "first_name='" + first_name + '\'' +
                ", last_name='" + last_name + '\'' +
                ", points=" + points +
                '}';
    }
}

File: Second Year/2nd Semester/Medii de Proiectare si Programare/C#/Laborator_1/domain/Participant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Laborator_1.domain
{
    public class Participant : Entity<int>
    {
        public string First_Name { get; set; }
        public string Last_Name { get; set; }
        public int Points { get; set; }

        public Participant(string first_Name, string last_Name, int points)
        {
            First_Name = first_Name;
            Last_Name = last_Name;
            Points = points;
        }

        public override string ToString()
        {
            return this.First_Name + ";" + this.Last_Name + ";" + this.Points.ToString();
        }
    }
}
